content/en/docs/rate-limits.md

```diff
@@ -40,11 +40,12 @@ can.
 
 Renewals are treated specially: they don't count against your **Certificates per
 Registered Domain** limit, but they are subject to a **Duplicate Certificate**
-limit of 5 per week. Note: renewals used to count against your Certificate per
-Registered Domain limit until March 2019, [but they don't
-anymore](https://community.letsencrypt.org/t/rate-limits-fixing-certs-per-name-rate-limit-order-of-operations-gotcha/88189).
+limit of 6 per week (168h) and 2 per day (24h) - the latter of which we implemented in June 2021 to warn users before they reach the weekly limit of 6.
 Exceeding the Duplicate Certificate limit is reported with the error message
-`too many certificates already issued for exact set of domains`.
+'too many certificates already issued for this exact set of domains'.
```

Comment:
I'd remove these lines entirely. This appears to be the only paragraph in the document where we provide (close to) the exact error message; there's no need for it at all.

Reply:
I think it's useful to have the exact (or something close to) error message, so web search engine can point here more precisely...